app/models/checklist_template_item.rb

```diff
@@ -21,6 +21,8 @@
 #
 class ChecklistTemplateItem < ApplicationRecord
   belongs_to :checklist_template
-  validates :name, :expected_duration_days, presence: true
   has_many :checklist_assignments, dependent: :destroy
+
+  validates :name, presence: true
+  validates :expected_duration_days, presence: true, numericality: {greater_than_or_equal_to: 0}
```

Comment:
Could you add a test please, there is a nice shoulda matcher we can use to make this happen fast :)

Reply:
sorry it took me so long to get around to this!